homeassistant/components/wilight/cover.py

```diff
@@ -0,0 +1,105 @@
+"""Support for WiLight Cover."""
+
+from pywilight.const import (
+    COVER_V1,
+    DOMAIN,
```

Comment:
We shouldn't import and use constants from the library in the Home Assistant specific code like eg `DOMAIN`. The library constants should only be used for the library interface.

Reply:
Done!

See: https://github.com/home-assistant/core/pull/46361